File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/41_compito_java_2022_11_10/Trainer.java
public class Trainer extends Dipendente {
	private boolean laurea;
	private String specialita;
	private float oreSettimanali;
	
	public Trainer(int codice, String tipo, String nomeDipendente, boolean laurea, String specialita, float oreSettimanali, float costoOrario) {
		super(codice, tipo, nomeDipendente, costoOrario);
		this.laurea = laurea;
		this.specialita = specialita;
		this.oreSettimanali = oreSettimanali;
	}
	
	public String toString() {
		return "tra" + "\t" + super.toString() + "\t" + laurea + "\t" + specialita + "\t" + oreSettimanali + "\t-\t-\t-\t" + costoOrario;
	}
}

File: Programming-languages-and-laboratory/Python/Exercises/41_compito_java_20221110/Trainer.py
from Dipendente import Dipendente


class Trainer(Dipendente):
    def __init__(self, codice, tipo, nome, laurea, specialita, ore_settimanali, costo_orario):
        super().__init__(codice, tipo, nome, costo_orario)
        self._laurea = laurea
        self._specialita = specialita
        self._ore_settimanali = ore_settimanali

    def __str__(self):
        return "tra" + "\t" + super().__str__() + "\t" + str(self._laurea) + "\t" + self._specialita + "\t" + str(self._ore_settimanali) + "\t-\t-\t-\t" + str(self._costo_orario)
